src/bufferutil.cc

```diff
@@ -105,9 +105,10 @@ NAN_METHOD(unmask) {
   }
 }
 
-NAN_MODULE_INIT(init) {
-  NAN_EXPORT(target, mask);
-  NAN_EXPORT(target, unmask);
+static Napi::Object Init(Napi::Env env, Napi::Object exports) {
```

Comment:
Call this `init` for consistency? or capitalize the other two.

Reply:
I see your point. I'll change it to ```init```.